### PR TITLE
Common_CreateCustomException

### PR DESCRIPTION
**Pull request by Maksim-Trolina**

### CODE REVIEW

File: Common/CustomException/ForbiddenException.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace CustomException
+{
+    /// <summary>
+    /// Indicates that the server understood the request, but refuses to authorize it
```
Comment (Cerberous007) [inline Common/CustomException/ForbiddenException.cs:6]: Not refuses to authorize
I think, there should be something like that: "user must be authorized"

**Review by ifelseelif — APPROVED** [review]

**Review by Cerberous007 — CHANGES_REQUESTED** [review]

File: Common/CustomException/ForbiddenException.cs (hunk 1 above)
Comment (Maksim-Trolina) [reply Common/CustomException/ForbiddenException.cs:6]: > Not refuses to authorize
> I think, there should be something like that: "user must be authorized"


So the translator translated, but the meaning is clear

**Review by LK777LK — APPROVED** [review]

**Review by nmamro — APPROVED** [review]

**Comment by nmamro** [comment]
Waiting for Dmitry

**Comment by nmamro** [comment]
And for resolving conflicts

File: Common/CustomException/ForbiddenException.cs (hunk 1 above)
Comment (Cerberous007) [reply Common/CustomException/ForbiddenException.cs:6]: Ok, may be my english isn`t so well))

**Review by Cerberous007 — APPROVED** [review]

File: Common/CustomException/ForbiddenException.cs (hunk 1 above)
Comment (strannick-rodion) [reply Common/CustomException/ForbiddenException.cs:6]: May be you should use not authorize or execute

**Pull request merged**